processing/src/main/java/io/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -73,7 +76,13 @@ public LookupReferencesManager(LookupConfig lookupConfig, final @Json ObjectMapp
   @LifecycleStart
   public void start()
   {
-    synchronized (lock) {
+    try {
```

Comment:
why do we need a read write lock, ? synchronized around a variable is enough ? is there any advantage ?